lbrynet/core/Wallet.py

```diff
@@ -1168,6 +1172,32 @@ def check_started():
             else:
                 network_start_d.errback(ValueError("Failed to connect to network."))
 
+        def get_cmd_runner():
+            self._cmd_runner = Commands(self.config, self.wallet, self.network)
+
+        def unlock(password):
+            if self._cmd_runner and self._cmd_runner.locked:
+                try:
+                    self._cmd_runner.unlock_wallet(password)
+                except InvalidPassword:
+                    log.warning("Incorrect password")
+                    check_locked()
+                    raise InvalidPassword
+            if self._cmd_runner.locked:
```

Comment:
why is this a separate if statement from the previous if statement? what if `self._cmd_runner` is None?